src/datasources/system/components/SystemQueryEditor.tsx

```diff
@@ -1,32 +1,30 @@
-import React, { ChangeEvent } from 'react';
-import { InlineField, Input } from '@grafana/ui';
+import React from 'react';
+import { InlineField, InlineFieldRow, RadioButtonGroup } from '@grafana/ui';
 import { QueryEditorProps } from '@grafana/data';
 import { SystemDataSource } from '../SystemDataSource';
-import { SystemQuery } from '../types';
+import { QueryType, SystemQuery } from '../types';
 
 type Props = QueryEditorProps<SystemDataSource, SystemQuery>;
 
+const QUERY_TYPES = [
+  {label: "Metadata", value: QueryType.Metadata},
+  {label: "Summary", value: QueryType.Summary}
+]
```

Comment:
It would nice if we had a generic utility method for generating this structure from an enum. Can you create that function in a new file `src/core/utils.ts`? The function signature will be:

```ts
export function enumToOptions(stringEnum: { [name: string]: string }): SelectableValue<string>
```

You should be able to get an array of name/value pairs from an enum using `Object.entries`, and then you can map that into to objects of `{ label, value }`.